src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java

```diff
@@ -2,6 +2,7 @@
 
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.json.JsonArray;
+import java.util.Objects;
```

Comment:
import by accident ?

Reply:
this is generated code, so it has been added by the generator

Reply:
yes thanks @vietj ;)
Do you think this PR is ok?

Reply:
yes, this code will actually be regenerated without it at some point as we removed the source of it :-)